### PR TITLE
added create category functionality

### PR DESCRIPTION
**Pull request by garyclayton02**
# Description
 This change allows a logged in user go to the Category List page.
While there, they can select the "Create Category" button. When they use the button, they will be directed to a page that will allow them to enter a new category name.
 After the user enters the category name, he can hit the "Save" button. This will save the category and direct them to the Category List page.

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Changes were made to CategoryForm in VS code, and to the CategoryController.cs code in Virtual Studio.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions

Please describe the tests required to verify your changes. Provide instructions so PR Tester can check functionality. Please also list any relevant details for your tests
- User needs to be logged in.
- User needs to go to the "Category" list page.
- User needs to click the "add new category" button.
- User needs to enter name for a new category.
- User will then need to be taken back to the "Category List" page for test to be a success.

# Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

File: Tabloid/client/src/components/tag/Tag.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import React, { useContext } from "react";
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faTrashAlt, faPencilAlt } from '@fortawesome/free-solid-svg-icons';
-import { TagContext } from "../../providers/TagProvider";
-import { useNavigate } from "react-router-dom"
+
+
```
Comment (andy-birt) [inline Tabloid/client/src/components/tag/Tag.js:4]: These imports need to stay in the file in order for your pr to function. However this file was before you started the pr just change it back to that and this should be good.

**Review by andy-birt — CHANGES_REQUESTED** [review]
See my comments. If you need help with any of it just let me know.

**Review by helloamandaball — CHANGES_REQUESTED** [review]
Missing some of your imports on the **Tag.js** file, which should have been part of the merged files in **[Main](https://github.com/NewForce-Cohort-5/tabloidfullstack-fool-s-spring/blob/main/Tabloid/client/src/components/tag/Tag.js):** 
```
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faTrashAlt } from '@fortawesome/free-solid-svg-icons';
import { TagContext } from "../../providers/TagProvider";
```

To install the Font Awesome Icon package in git terminal. Navigate to where your package.json file exists in your client folder: `~{working-directory}/tabloidfullstack-fool-s-spring/Tabloid/client, install these lines of code:
```
npm i --save @fortawesome/fontawesome-svg-core
npm install --save @fortawesome/free-solid-svg-icons
npm install --save @fortawesome/react-fontawesome
```

**Review by andy-birt — APPROVED** [review]
🔥 🔥 Awesome work sir! 🔥 🔥

**Pull request merged**